src/test-helper.js

```diff
@@ -0,0 +1,691 @@
+/**
```

Comment:
The contents of this file are identical to the previous revision of `etc/test-helper.js`.